kasa/aestransport.py

```diff
@@ -247,19 +280,14 @@ async def send(self, request: str):
         if self.needs_login:
             raise SmartDeviceException("Login must be complete before trying to send")
```

Comment:
Nothing was changed in this PR, but I'm wondering if we should just execute automatic login here?

Reply:
At one point I thought the transport needed the `terminal_id` and `request_id` from the protocol to do the login because that's what plugP100 does but I have seen other libraries that seem not to do this.  I'll investigate for a later PR if that's ok because I'd need to somehow get hold of another AES device to test.